test/fixtures/peers.js

```diff
@@ -51,7 +51,7 @@ const Peer = stampit({
 	},
 	init({ broadhash, nonce, state }) {
 		this.dappid = null;
-		this.height = faker.random.number({ max: 9999, min: 1 });
+		this.height = _.sample([50, 70, 90, 110]);
```

Comment:
In our application, do we store height as a number or as a string?

Reply:
swagger.yml:
```
      height:
        type: integer
        example: 123
        minimum: 1
```

Reply:
we keep height as an integer when we do the group by `height` using lodash it returns the string, so it's about changing that string to the integer. In the application, we always have the integer for both `height` and `networkHeight`.

Reply:
For clarity, we should `parseInt` here then.